crates/ruff_linter/src/fix/edits.rs

```diff
@@ -151,24 +151,24 @@ pub(crate) fn add_to_dunder_all<'a>(
     stylist: &Stylist,
 ) -> Vec<Edit> {
     let (insertion_point, export_prefix_length) = match expr {
-        Expr::List(ExprList { elts, range, .. }) => (
+        Expr::List(ExprList { elts, .. }) => (
             elts.last()
-                .map_or(range.end() - "]".text_len(), Ranged::end),
+                .map_or_else(|| expr.end() - "]".text_len(), Ranged::end),
```

Comment:
I'm not so sure if changing from` map_or` to `map_or_else` is worth here, considering that the computation is fairly cheap (it subtracts -1). A function call would be n-times more expensive than the computation we do here.

Reply:
Yeah, debatable I guess. I'll revert this change.